traffic_portal/app/src/modules/public/login/login.tpl.html

```diff
@@ -21,12 +21,13 @@
     <div class="col-md-12 col-sm-12 col-xs-12">
         <div class="x_panel">
             <div class="x_title">
-                <h2>Login</h2>
+                <h2 ng-show="!oAuthEnabled">Login</h2>
+                <h2 ng-show="oAuthEnabled">Login With Single Sign On</h2>
                 <div class="clearfix"></div>
             </div>
             <div class="x_content">
                 <br>
-                <form name="loginForm" role="form" class="form-horizontal form-label-left" novalidate>
+                <form name="loginForm" role="form" class="form-horizontal form-label-left" novalidate ng-show="!oAuthEnabled">
```

Comment:
Same as above RE: `ng-if` vs `ng-show`.

Also, why are we doing `novalidate` on these forms? Your change is consistent with what we have already, but we aren't doing any angular validations on these forms, so `novalidate` only disables validations that aren't being overridden anyway.

Reply:
done for the ng-if
as for the novalidate, is this worth updating now or should i stay with going for consistency with the other code?

Reply:
unless you wanna change both - which you absolutely do not have to do -, don't bother